test-cloud/pom.xml

```diff
@@ -104,11 +104,24 @@
     <module>test-cloud-optaweb</module>
     <module>test-cloud-performance</module>
     <module>test-cloud-ha-cep</module>
+    <module>springboot</module>
```

Comment:
Maybe it's worth following the same naming convention for this new `springboot` Maven project, and rename it to something like `test-cloud-springboot`? don't have a strong opinion on this, up to you.